src/authentication/index.js

```diff
@@ -546,6 +546,28 @@ Authentication.prototype.userInfo = function(accessToken, cb) {
     .end(responseHandler(cb, { ignoreCasing: true }));
 };
 
+/**
+ * Makes a call to the `/usernamepassword/challenge` endpoint
+ * and returns the challenge (captcha) if necessary.
+ *
+ * @method getChallenge
+ * @param {callback} cb
+ */
+Authentication.prototype.getChallenge = function(cb) {
+  assert.check(cb, { type: 'function', message: 'cb parameter is not valid' });
+
+  if (!this.baseOptions.state) {
+    return cb();
```

Comment:
Probably this would never be hit, as a `state` is normally set for every request. Is this intended as backward compatibility support? and, will it match the "challenge not necessary" object format?

Reply:
When lock is hosted in the application (ie, auth happens in `/co/authenticate`), there is no internal state.